src/Message.js

```diff
@@ -82,7 +82,7 @@ export default class Message extends React.Component {
       <View>
         {this.renderDay()}
         <View style={[styles[this.props.position].container, {
-          marginBottom: this.isSameUser(this.props.currentMessage, this.props.nextMessage) ? 2 : 10,
+          marginBottom: this.isSameUser(this.props.currentMessage, this.props.nextMessage) ? this.props.marginSameUser : this.props.marginDifferentUser,
```

Comment:
Hi @jayshah123 

I think that this is not related to the original PR. In my opinion, is better to split this changes in two different PR.